lib/skate/application.ex

```diff
@@ -37,9 +37,9 @@ defmodule Skate.Application do
           []
         end ++
         [
+          Skate.Migrate,
```

Comment:
I can see that _maybe_ this needs to be done before the Endpoint is started, but does it need to come before the PubSub?

Also, _when_ would code in the Endpoint conflict with the Migrations? I assume this would never happen in a deployment due to our health checks?

Reply:
Basically I was thinking that code in the Endpoint would refer to some column in the database that's introduced in a migration. That'll cause an error in the 

Do the healthchecks fail if there are any un-run migrations?

My reason for this was remembering that a while ago, when I added a migration, I was asked to pull the migration out to a pre-PR that got deployed before the main PR was merged. Maybe that wasn't necessary though?